SwiftSampleProject/SwiftyVokoder/Models/Station+CoreDataProperties.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Station+CoreDataProperties.swift
+//  
+//
+//  Created by Carl Hill-Popper on 11/13/15.
+//  Copyright © 2015 Vokal.
+//
+//  Choose "Create NSManagedObject Subclass…" from the Core Data editor menu
+//  to delete and recreate this implementation file for your updated model.
+//
+
+import Foundation
+import CoreData
+
+extension Station {
```

Comment:
These are the auto generated "machine" files from Xcode's Core Data model editor.  Thus the extension and the lack of project I guess.